src/python/grpcio/grpc/_cython/_cygrpc/records.pyx.pxi

```diff
@@ -255,10 +255,13 @@ cdef class ByteBuffer:
 
   def __cinit__(self, bytes data):
     grpc_init()
+    self._own_buffer = True
```

Comment:
Similar comment here as in C++.  Is there any case where this attribute will actually be true?

Reply:
This would end up being true for write ops.  This is no longer used in the current implementation however.